pymongoexplain/commands.py

```diff
@@ -20,15 +20,20 @@
 
 from bson.son import SON
 from pymongo.collection import Collection
+from pymongo.helpers import _index_document, _fields_list_to_dict
```

Comment:
Even though we maintain pymongo we can't rely on any private methods (in python names starting with `_` mean private). Could you vendor (copy) these two methods instead?

Reply:
Done.